Sparkle/SUGuidedPackageInstaller.m

```diff
@@ -55,11 +55,11 @@ + (AuthorizationRef)authorizationForExecutable:(NSString*)executablePath
 		const char* executableFileSystemRepresentation = [executablePath fileSystemRepresentation];
 		
 		// Prepare a right allowing script to execute with privileges
-		AuthorizationItem right;
-		memset(&right,0,sizeof(right));
-		right.name = kAuthorizationRightExecute;
-		right.value = (void*) executableFileSystemRepresentation;
-		right.valueLength = strlen(executableFileSystemRepresentation);
+        AuthorizationItem right = {
```

Comment:
It fails on the new `__nonnull` attribute. I'll shim it with an `ifdef`ed macro.

> /Users/travis/build/sparkle-project/Sparkle/Sparkle/Autoupdate/Autoupdate.m:154:35: error: expected ')'
        `[statusCtl setButtonTitle:SULocalizedString(@"Cancel Update", @"") target:nil action:Nil isDefault:NO];`